Cargo.toml

```diff
@@ -57,7 +65,7 @@ strsim = "0.10"
 structopt = "0.3.21"
 toml = "0.5"
 tracing = "0.1.26"
-url = "2.2.2"
```

Comment:
i think compiling with `--no-default-features` made this break, so had to add it explicitly so our no-composition-js would compile